[company_addresses,84]Add resources and methods for admin company add…

almabud2 opened the pull request:
…resses.

- Add a new resource called `APIAdminCompanyAddress`
- Add the resource `APIAdminCompanyAddress` in the APIAdminCompany

Fixed #84

almabud2 committed:
[company_addresses,84]Add resources and methods for admin company addresses.
- Add a new resource called `APIAdminCompanyAddress`
- Add the resource `APIAdminCompanyAddress` in the APIAdminCompany

Fixed #84

OdinsPlasmaRifle commented on rehive/api/resources/admin_resources.py:
```diff
@@ -618,3 +621,12 @@ def register(self,
         return response
+
+
+class APIAdminCompanyAddress(ResourceList):
```
Why is this a `ResourceList`. there can only be 1 resource returned on the `/admin/company/address/` endpoint. I think this should instead inherit from `Resource`.

OdinsPlasmaRifle reviewed (changes requested):
One small fix.

almabud2 committed:
[company_addresses,84] Do requested changes.
- Remove Resource class and add only Resource in the APIAdminCompanyAddress

Fixed #84
